docker-compose.yml

```diff
@@ -68,6 +69,7 @@ services:
       - "30106:30106"
       - "8080:8080"
     labels:
+      - "io.balena.features.balena-api=1"
```

Comment:
these should be `io.balena.features.supervisor-api` instead of balena-api

then the curl error mentioned by @ketilmo is fixed

Reply:
Nice catch, thank you for checking, let me fix it